scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/JobsPaginationController.java

```diff
@@ -75,20 +75,18 @@ public void lastPage() {
 
     @Override
     public void nextPage() {
-        int curPage = model.getPage();
-        if (curPage == model.getMaxPage())
+        if (!this.hasNext())
```

Comment:
could you use curly brackets after if?
i think it is really common practice in java + it is java code convention